tests/server/test_query_params.py

```diff
@@ -305,28 +305,37 @@ def test_page_limit_max(self):
             expected_detail=f"Max allowed page_limit is {CONFIG.page_limit_max}, you requested {CONFIG.page_limit_max + 1}",
         )
 
-    def test_list_has_all(self):
-        request = '/structures?filter=elements HAS ALL "Ba","F","H","Mn","O","Re","Si"'
+    def test_value_list_operator(self):
+        request = f"/structures?filter=dimension_types HAS < 1"
         self._check_error_response(
-            request, expected_status=501, expected_title="NotImplementedError"
+            request,
+            expected_status=501,
+            expected_title="NotImplementedError",
+            expected_detail="set_op_rhs not implemented for [Token(HAS, 'HAS'), Token(OPERATOR, '<'), 1].",
```

Comment:
If you update the error message as suggested above, this should be updated as well.